src/twisted/web/_http2.py

```diff
@@ -103,11 +103,23 @@ class H2Connection(Protocol, TimeoutMixin):
     @ivar _sender: A handle to the data-sending loop, allowing it to be
         terminated if needed.
     @type _sender: L{twisted.internet.task.LoopingCall}
+
+    @ivar abortTimeout: The number of seconds to wait after we attempt to shut
```

Comment:
(Even if H2Connection itself isn't exposed - I'm not sure that it is - it would be nice to be consistent with the surrounding variables, if there's no need for it to be public.)